### PR TITLE
Add new client for Buf/Protocol Buffers

### PR DESCRIPTION
**Pull request by suzuki**
## About

Current `lsp-bufls` using the [Buf Language Server](https://github.com/bufbuild/buf-language-server). But that LSP server was already archived project.
The `buf` CLI was supported LSP server from [v1.43.0](https://github.com/bufbuild/buf/releases/tag/v1.43.0). So I want to try to use it.

It's a first time to add new client for me. please let me know if I need anything else.

### CODE REVIEW

**Review by jcs090218 — APPROVED** [review]

**Comment by jcs090218** [comment]
Can you add this to the CHANGELOG.org? Thanks!

**Comment by suzuki** [comment]
I added this to CHANGELOG.

**Review by jcs090218 — APPROVED** [review]

File: clients/lsp-bufls.el [hunk 1]
```diff
@@ -63,6 +64,41 @@
...
+               (> (length lsp-buf-args) 0))
+      (setq args (append args lsp-buf-args)))
+    args))
+
+(lsp-register-client
+ (make-lsp-client :new-connection (lsp-stdio-connection
+                                   #'lsp-buf--stdio-command)
+                  :activation-fn (lsp-activate-on "protobuf")
+                  :language-id "protobuf"
+                  :priority -1
```
Comment (jcs090218) [inline clients/lsp-bufls.el:99]: Maybe we should make this 0 and replace the deprecated bufls server? 🤔

Comment (suzuki) [reply clients/lsp-bufls.el]: I tried to change priorities by https://github.com/emacs-lsp/lsp-mode/pull/4580/commits/192cc095ad811a139b6b5a638f26c37dcbb6b8d6
Does this fit your idea?

Comment (jcs090218) [reply clients/lsp-bufls.el]: That works! Thanks! :)

**Review by jcs090218 — APPROVED** [review]

**Pull request merged**